docs/guides/universal-profile/01-create-profile.md

```diff
@@ -284,41 +287,42 @@ async function createUniversalProfile() {
 }
 
 createUniversalProfile();
-// {
-//   ERC725Account: {
-//     address: '0x...',
-//     receipt: {
-//       to: null,
-//       from: '0x...',
-//       contractAddress: '0x...',
-//       transactionIndex: 0,
-//       gasUsed: [BigNumber],
-//       logsBloom: '0x...',
-//       blockHash: '0x...',
-//       transactionHash: '0x...',
-//       logs: [],
-//       blockNumber: ...,
-//       confirmations: 1,
-//       cumulativeGasUsed: [BigNumber],
-//       status: 1,
-//       type: 0,
-//       byzantium: true,
-//       events: []
-//     }
-//   },
-//   KeyManager: {
-//     address: '0x646e989A0840CE4c3bac39d535Af736db2371107',
-//     receipt: {
-//       ...
-//     }
-//   },
-//   UniversalReceiverDelegate: {
-//     address: '0xc2f51A2891E7c78541a2CADEff9146F1E0E13E2a',
-//     receipt: {
-//       ...
-//     }
-//   }
-// }
+
+> {
+  ERC725Account: {
+    address: '0x...',
```

Comment:
```js
/**
...
*/
```


This allows better color formatting as it will show as a comment and not with weird colors.